tests/appsec/iast/test_vulnerability_schema.py

```diff
@@ -0,0 +1,21 @@
+import jsonschema
+import json
+from pathlib import Path
+from utils import interfaces, features
+
+
+@features.iast_schema
+class TestIastVulnerabilitySchema:
+    def test_vulnerability_schema(self):
+        schema_path = Path(__file__).parent / "vulnerability_schema.json"
+        with open(schema_path, "r") as f:
+            schema = json.load(f)
+        validator = jsonschema.Draft7Validator(schema)
+        spans = [s for _, s in interfaces.library.get_root_spans()]
```

Comment:
If I am understanding this correctly... this test rely on the test order, this should be the last executed test.

* If this is the first test executed, will success without any error _(spans will be empty)_
* If there is no root spans with vulnerabilities, `if "_dd.iast.json" not in meta:` will be always `true` and test will success.

Can we somehow, force this test to be latest one in the scenario?

Reply:
@uurien Because `setup` phase for all tests runs before any `test` phase, order does not matter. By the team this test runs, all setup phases will have run already, and weblog will already have been stopped.

Reply:
ah true! thanks for the clarification!